maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/test/java/org/apache/maven/scm/provider/git/jgit/command/checkin/JGitCheckInCommandCommitterAuthorTckTest.java

```diff
@@ -54,7 +60,7 @@
 {
     
     @Override
-    protected void setUp()
+    public void setUp()
```

Comment:
Apparently, this still works (at least, no tests seem to be failing because this doesn't run) but wouldn't it be more nice to annotate with `@Before` (in JUnit 4 style)?

Reply:
fixed